### DISCUSSION TITLE
Default login password not working.

### DISCUSSION THREAD

**Discussion by salvinoto** [andreapollastri/cipi#454] [Q&A]
I just completed a fresh installation of cipi using the automatic installer, and I follow the instructions and saved all the passwords at the end. Turns out, I can't use administrator / 12345678, it just gives me red exclamations when trying to sign in, which I'm assuming means incorrect username and password. Is there anything else it can be?

**Comment by trueriders** [comment]
I have the same problem!

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
